cms/release_calendar/forms.py

```diff
@@ -0,0 +1,82 @@
+from datetime import datetime
+from typing import Any
+
+from django.core.exceptions import ValidationError
+from django.utils.translation import gettext as _
+from wagtail.admin.forms import WagtailAdminPageForm
+
+from .enums import NON_PROVISIONAL_STATUS_CHOICES, ReleaseStatus
+
+
+class ReleaseCalendarPageAdminForm(WagtailAdminPageForm):
+    """Convenience administrative form for easier calendar page data validation."""
+
+    def __init__(self, *args: Any, **kwargs: Any) -> None:
+        super().__init__(*args, **kwargs)
+
+        if self.instance.status != ReleaseStatus.PROVISIONAL:
+            # Once a release calendar page is confirmed, it cannot go back to provisional
+            self.fields["status"].choices = NON_PROVISIONAL_STATUS_CHOICES
+
+        if self.instance.status == ReleaseStatus.PUBLISHED:
```

Comment:
Q: Is there any reason why equality is chosen over identity for enums? I expected `self.instance.status is ReleaseStatus.PUBLISHED`

Reply:
Ah I though it resolved back to enums not strings. 👍